app/javascript/pulfalight/pulfalight_loader.es6

```diff
@@ -45,6 +46,17 @@ export default class {
     toc.build()
   }
 
+  setup_toc_collapse() {
+    const mediaQuery = window.matchMedia('(max-width: 768px)')
```

Comment:
Where does 768 come from? If we can't use a variable defined somewhere else, let's add a comment pointing to where it comes from.

Reply:
I'll add a comment referencing the file containing the breakpoints for the application. I'll make a ticket since this seems like an inconsistency throughout the repo.